edx-platform/pearson-vue-theme/lms/student-account/login.underscore

```diff
@@ -1,80 +1,70 @@
-<div class="js-form-feedback" aria-live="assertive" tabindex="-1">
+<div class="background-login"></div>
+
+<div>
+  <img class="signin-logo" src="/static/pearson-vue-theme/images/CertPREP_Logo.png">
```

Comment:
Would this need to be changed via /admin? @luismorenolopera

Reply:
@Squirrel18 maybe, but it is not the intention of this PR